src/main/webapp/js/lib/calendar.js

```diff
@@ -17,6 +17,10 @@
 // TODO(#34): Add ability to choose which section the user wants, not just
 //  automatically choosing the first section
 
+export const Calendar = (() => {
```

Comment:
While we're here, let's do:

1. No need for an anonymous function
2. exports belong at the bottom of the file
3. No need for this to be a named export either

So I think this should be:

```
export default Calendar = {
  addCourse: addCourse,
  initCalendar: initCalendar
};
```

Reply:
actually I think this is probably fine too:

```
export default {
  addCourse: addCourse,
  initCalendar: initCalendar
};
```